app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -957,6 +966,33 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
         }
     }
 
+    private fun hasCameraPermission(context: Context): Boolean {
+        return AndroidPermissionsHelper.checkPermission(context, android.Manifest.permission.CAMERA)
+    }
+
+    private fun getCameraPermission() {
+        requestPermissions(
```

Comment:
The same way you did above, you could did here. 

Use `AndroidPermissionsHelper.requestPermission(...)`

or, I'd move it to `AndroidPermissionsHelper` class in order to centralize all the permissions stuffs.